File: Zadanie1 github/C#/Zadanie2.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Wybierz jednostkę wejściową (C lub F): ");
        string opcja = Console.ReadLine().ToUpper();

        Console.Write("Podaj wartość temperatury: ");
        double temp = Convert.ToDouble(Console.ReadLine());

        if (opcja == "C")
        {
            double wynik = temp * 1.8 + 32;
            Console.WriteLine($"{temp}°C to {wynik:F2}°F");
        }
        else if (opcja == "F")
        {
            double wynik = (temp - 32) / 1.8;
            Console.WriteLine($"{temp}°F to {wynik:F2}°C");
        }
        else
        {
            Console.WriteLine("Nieprawidłowy wybór.");
        }
    }
}

File: Zadanie1 github/Python/Zadanie2.py

opcja = input("Wybierz jednostkę wejściową (C lub F): ").upper()
temp = float(input("Podaj wartość temperatury: "))

if opcja == "C":
    wynik = temp * 1.8 + 32
    print(f"{temp}°C to {wynik:.2f}°F")
elif opcja == "F":
    wynik = (temp - 32) / 1.8
    print(f"{temp}°F to {wynik:.2f}°C")
else:
    print("Nieprawidłowy wybór. Wybierz C lub F.")